coq-unimath.opam

```
@@ -0,0 +1,14 @@
+opam-version: "2.0"
+maintainer: "The UniMath Development Team"
+homepage: "https://github.com/UniMath/UniMath"
+dev-repo: "git+https://github.com/UniMath/UniMath.git"
+bug-reports: "https://github.com/UniMath/UniMath/issues"
+license: "Kind of MIT"
```

I suggest the following slight reformulation:
```suggestion
license: "Similar to MIT license"
```